conda/psyclone/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "psyclone" %}
-{% set version = "2.5.0" %}
+{% set version = "3.0.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: https://github.com/stfc/PSyclone/archive/refs/tags/{{version}}.tar.gz
-  sha256: 00f5d4bed877293bd6327e1e545532a01ef288687f2e88b4f53799191bb819c3
+  url: https://github.com/stfc/PSyclone/archive/refs/tags/v{{version}}.tar.gz
```

Comment:
While I don't care, I seem to remember that spack prefers pypi package url over github.  If you think it makes sense to change, details including sha256 are here:
https://pypi.org/project/PSyclone/3.0.0/#psyclone-3.0.0.tar.gz